testing/src/test/java/org/motechproject/nms/testing/it/kilkari/CsrServiceBundleIT.java

```diff
@@ -339,6 +339,58 @@ public void verifySubscriptionCompletion() {
         assertEquals(3, subscriptions.size());
     }
 
+
+    @Test
+    public void verifyFT140() {
```

Comment:
Agreed -- either the test case should have a descriptive name and a comment with the id #, or the test can be named with the id # but should have a descriptive comment. 

-----Original Message-----
From: "kosh" <notifications@github.com>
Sent: ‎6/‎2/‎2015 10:52 AM
To: "motech-implementations/mim" <mim@noreply.github.com>
Subject: Re: [mim] FT140 (#293)

In testing/src/test/java/org/motechproject/nms/testing/it/kilkari/CsrServiceBundleIT.java:
> @@ -339,6 +339,58 @@ public void verifySubscriptionCompletion() {
>          assertEquals(3, subscriptions.size());
>      }
>  
> +
> +    @Test
> +    public void verifyFT140() {
I think if we are sticking with the test#id for the naming, that we have a short description for what the test is trying to do (instead of cross-referencing the code and spec all the time)
—
Reply to this email directly or view it on GitHub.